src/huggingface_hub/inference/_client.py

```diff
@@ -1309,3 +1318,52 @@ def _resolve_url(self, model: Optional[str] = None, task: Optional[str] = None)
             # Otherwise, we use the default endpoint
             else f"{INFERENCE_ENDPOINT}/models/{model}"
         )
+
+    def get_model_status(self, model: Optional[str] = None, *, token: Optional[str] = None) -> ModelStatus:
+        """
+            A function which returns the status os a specific model, from the huggingface Inference API.
+
+            Parameters:
+            -----------
+            model -> Optional[str], default=None
+                Identifier of the model for witch the status gonna be returned(retrieve). If model is not provided, 
+                the model associated with this instance of InferenceClient will be used. The 
+                identifier should not be a URL.
+
+            token -> Optional[str], default=None
+                 Optional token for authenticating with the Hugging Face API. If not provided, the 
+                 token associated with this instance of InferenceClient will be used.
+            
+                 
+            Raises
+            ------
+            ValueError
+                If the model is missing, meaning is not provided. Or the provided model is a URL. And finally
+                it raises an error if the API returns an error message(missing model).
+            
+                
+            Returns
+            -------
+            ModelStatus
+                An instance of ModelStatus dataclass, containing information about the state of 
+                the model: load, state, compute type and framework.
+        """
```

Comment:
Thanks for the effort to write the docstring @sifisKoen! May I ask to refactor it to make comply with the existing docstring format used in `huggingface_hub`. [Here is an example](https://github.com/huggingface/huggingface_hub/blob/4f1e4bbb28b58ffbc2588d2c549c6be0d083bac8/src/huggingface_hub/inference/_client.py#L175) with the "args", "returns" and "raises" sections. This is important to generate the documentation of the library.

Reply:
Ok cool I will just make the changes so this doc string will follow your format.